clustered/server/src/main/java/org/ehcache/clustered/server/ClusterTierManagerActiveEntity.java

```diff
@@ -100,14 +95,16 @@ public void addStateTo(StateDumpCollector dump) {
     {
       Map<ClientDescriptor, ClientState> clients = new HashMap<>(clientStateMap);
       dump.addState("clientCount", String.valueOf(clients.size()));
-      StateDumpCollector clientsDump = dump.subStateDumpCollector("clients");
-      int idx = 0;
+
+      List<Map> allClients = new ArrayList<>();
```

Comment:
yes I guess that would make more sense.  I will change it to use the ArrayList size.